docker-compose.yml

```diff
@@ -114,14 +115,6 @@ services:
       stripe:
         condition: service_started
 
-  tests:
-    image: warehouse:docker-compose
-    pull_policy: never
-    volumes: *base_volumes
-    depends_on:
-      stripe:
-        condition: service_started
```

Comment:
I think there's still value in having this split out to it's own compose service, if nothing else to prevent needing to share `dev/environment and redis servisce for tests. Not a blocker for now, but a good testing improvement for the future.